### PR TITLE
configure timestamper plugin in jenkins

### PR DESCRIPTION
**Pull request by estute**
Configuration Pull Request
---

Make sure that the following steps are done before merging:

  - [ ] A DevOps team member has approved the PR if it is code shared across multiple services and you don't own all of the services.
  - [ ] Are you adding any new default values that need to be overridden when this change goes live? If so:
    - [ ] Update the appropriate internal repo (be sure to update for all our environments)
    - [ ] If you are updating a secure value rather than an internal one, file a DEVOPS ticket with details.
    - [ ] Add an entry to the CHANGELOG.
  - [ ] If you are making a complicated change, have you performed the proper testing specified on the [Ops Ansible Testing Checklist](https://openedx.atlassian.net/wiki/display/EdxOps/Ops+Ansible+Testing+Checklist)?  Adding a new variable does not require the full list (although testing on a sandbox is a great idea to ensure it links with your downstream code changes).
  - [ ] Think about how this change will affect Open edX operators.  Have you updated the wiki page for the next Open edX release?

### CODE REVIEW

File: playbooks/roles/jenkins_common/defaults/main.yml [hunk 1]
```diff
@@ -23,7 +23,7 @@ jenkins_common_debian_pkgs:
 jenkins_common_python_versions: []
 
 jenkins_common_configuration_git_url: https://github.com/edx/jenkins-configuration.git
-jenkins_common_jenkins_configuration_branch: master
+jenkins_common_jenkins_configuration_branch: 'estute/te-2947'
```
Comment (estute) [inline playbooks/roles/jenkins_common/defaults/main.yml]: I plan on changing this back to `master` once https://github.com/edx/jenkins-configuration/pull/136 has been merged in. This is just for testing.

File: playbooks/roles/jenkins_common/defaults/main.yml [hunk 2]
```diff
@@ -289,3 +290,8 @@ JENKINS_SPLUNK_HOSTNAME: ''
 JENKINS_SPLUNK_PORT: 8088
 JENKINS_SPLUNK_APP_URL: ''
 JENKINS_SPLUNK_TOKEN: ''
+
+# timestamper
+jenkins_common_timestamper_system_clock_format: ''
+jenkins_common_timestamper_elapsed_time_format: ''
+jenkins_common_timestamper_enabled_on_pipelines: 'false'
```
Comment (jmbowman) [inline playbooks/roles/jenkins_common/defaults/main.yml]: Is there a good reason not to default to the values for build Jenkins above?

**Review by jmbowman — APPROVED** [review]
One question, otherwise 👍

File: playbooks/roles/jenkins_common/defaults/main.yml (hunk 2 above)
Comment (estute) [reply playbooks/roles/jenkins_common/defaults/main.yml]: No, that's a fair point. Fixing.

**Pull request merged**